modules/jaeger-sidecar-manual.adoc

```diff
@@ -0,0 +1,57 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-deploying.adoc
+////
+
+[id="jaeger-sidecar-manual_{context}"]
+= Manually injecting sidecars
+:pantheon-module-type: PROCEDURE
+
+For controller types other than `Deployments` (for example,  `StatefulSets`, `DaemonSets`, etc.), you can manually define the Jaeger Agent sidecar in your specification.
+
+The following snippet shows the manual definition you can include in your containers section for a Jaeger Agent sidecar:
+
+.example sidecar definition for a StatefulSet
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: example-statefulset
+  namespace: example-ns
+  labels:
+    app: example-app
+spec:
+
+    spec:
+      containers:
+        - name: example-app
+          image: jaegertracing/vertx-create-span:operator-e2e-tests
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+        - name: jaeger-agent
+          image: jaegertracing/jaeger-agent:1.17.0  # The agent version must match the operator version
```

Comment:
This would need to be `image: registry.redhat.io/distributed-tracing/jaeger-agent-rhel7:<version>`

Reply:
Just realised - this is going to be an issue for disconnected environments where images are referenced by SHAs.

Reply:
Added disconnected installs to my To Do list for next release.